homeassistant/components/api_streams/sensor.py

```diff
@@ -88,3 +46,13 @@ def state(self):
     def unit_of_measurement(self):
         """Return the unit of measurement."""
         return "clients"
+
+    @callback
+    def _increment(self, connection):
```

Comment:
typo in method name.

I agreed with balloob that connection should not be in the method parameter. It is unused anyway.

Reply:
What's the typo?

Reply:
Sorry, it is not :sweat_smile: